### PR TITLE
paulczar fixes plus example file

### PR DESCRIPTION
**Pull request by chanezon**
Signed-off-by: Patrick Chanezon <patlist@chanezon.com>
Includes @paulczar https://github.com/docker/compose/compare/master...paulczar:beginner_doc plus an example yml file at the top

### CODE REVIEW

**Comment by chanezon** [comment]
To fix #1148

File: docs/index.md [hunk 1]
```diff
@@ -108,13 +102,18 @@ specify how to build the image using a file called
...
+This tells Docker to:
 
-This tells Docker to include Python, your code, and your Python dependencies in
-a Docker image. For more information on how to write Dockerfiles, see the
-[Docker user
-guide](https://docs.docker.com/userguide/dockerimages/#building-an-image-from-a-dockerfile)
-and the
-[Dockerfile reference](http://docs.docker.com/reference/builder/).
+* Build an image starting with the Python 2.7 image.
+* Add the curret directory `.` into the path `/code` in the image.
```
Comment (aanand) [inline docs/index.md]: Typo ("curret")

File: docs/index.md [hunk 2]
```diff
@@ -108,13 +102,18 @@ specify how to build the image using a file called
     ADD . /code
     WORKDIR /code
     RUN pip install -r requirements.txt
+    CMD python app.py
```
Comment (aanand) [inline docs/index.md]: Good idea.

File: docs/index.md [hunk 3]
```diff
@@ -15,34 +15,28 @@ recommend that you use it in production yet.
...
+3. Lastly, run `docker-compose up` and Compose will start and run your entire app.
+
+A `docker-compose.yml` looks like this:
 
 ```yaml
 web:
   build: .
-  links:
-   - db
+  command: python app.py
```
Comment (aanand) [inline docs/index.md]: Since CMD is in the Dockerfile, we don't need this.

File: docs/index.md [hunk 4]
```diff
@@ -134,19 +133,21 @@ Next, define a set of services using `docker-compose.yml`:
```
Comment (aanand) [inline docs/index.md]: We can also remove `command:` here.

File: docs/index.md [hunk 5]
```diff
@@ -134,19 +133,21 @@ Next, define a set of services using `docker-compose.yml`:
...
-   port 5000 on the container to port 5000 on the host machine, connect up the
-   Redis service, and mount the current directory inside the container so we can
-   work on code without having to rebuild the image.
- - `redis`, which uses the public image
-   [redis](https://registry.hub.docker.com/_/redis/), which gets pulled from the
-   Docker Hub registry.
+#### web
+
+* Builds from the `Dockerfile` in the current directory. 
+* Defines to run the command `python app.py` inside the image on start.
```
Comment (aanand) [inline docs/index.md]: This can go if we remove `command:`.

File: docs/index.md [hunk 6]
```diff
@@ -108,13 +102,18 @@ specify how to build the image using a file called
...
-This tells Docker to include Python, your code, and your Python dependencies in
-a Docker image. For more information on how to write Dockerfiles, see the
-[Docker user
-guide](https://docs.docker.com/userguide/dockerimages/#building-an-image-from-a-dockerfile)
-and the
-[Dockerfile reference](http://docs.docker.com/reference/builder/).
+* Build an image starting with the Python 2.7 image.
+* Add the curret directory `.` into the path `/code` in the image.
+* Set the working directory to `/code`.
+* Install your Python dependencies.
```
Comment (aanand) [inline docs/index.md]: One more: "Set the default command for the container to `python app.py`".

**Comment by aanand** [comment]
Nice. Could you also implement @fredlf's suggestions from https://github.com/docker/compose/pull/1162?

**Comment by chanezon** [comment]
@aanand done.

**Comment by aanand** [comment]
LGTM

**Comment by aanand** [comment]
ping @bfirsh @dnephin @moxiegirl

**Comment by bfirsh** [comment]
LGTM

@aanand We should probably also update README.md :)

**Comment by bfirsh** [comment]
Thanks @chanezon!

**Pull request merged**